great_expectations/render/renderer/column_section_renderer.py

```diff
@@ -41,15 +41,25 @@ def _get_column_name(cls, ge_object):
 
 class ProfilingResultsColumnSectionRenderer(ColumnSectionRenderer):
 
-    def __init__(self, overview_table_renderer=None):
+    def __init__(self, overview_table_renderer=None, expectation_string_renderer=None):
         if overview_table_renderer is None:
             overview_table_renderer = {
                 "class_name": "ProfilingOverviewTableContentBlockRenderer"
             }
+        if expectation_string_renderer is None:
+            expectation_string_renderer = {
+                "class_name": "ExpectationStringRenderer"
+            }
         self._overview_table_renderer = load_class(
             class_name=overview_table_renderer.get("class_name"),
             module_name=overview_table_renderer.get("module_name", "great_expectations.render.renderer.content_block")
         )
+        self._expectation_string_renderer = load_class(
+            class_name=expectation_string_renderer.get("class_name"),
+            module_name=expectation_string_renderer.get(
+                "module_name", "great_expectations.render.renderer.content_block"
+            )
+        )
```

Comment:
Can you change `load_class` (which is giving the class object back) to `instantiate_class_from_config` (which gives an instance back)? That also should allow the user to configure the expectation_string_renderer if desired.